docs/pages/toolchain/katana/genesis.md

```diff
@@ -1,178 +1,95 @@
-	"parentHash": "0x999",
-	"timestamp": 5123512314,
-	"stateRoot": "0x99",
-	"sequencerAddress": "0x100",
-	"gasPrices": {
-		"ETH": 1111,
-		"STRK": 2222
-	},
-	"feeToken": {
-		"address": "0x55",
-		"name": "ETHER",
-		"symbol": "ETH",
-		"decimals": 18,
-		"class": "0x8",
-		"storage": {
-			"0x111": "0x1",
-			"0x222": "0x2"
-		}
-	},
-	"universalDeployer": {
-		"address": "0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02bf",
-		"storage": {
-			"0x10": "0x100"
-		}
-	},
-	"accounts": {
-		"0x66efb28ac62686966ae85095ff3a772e014e7fbf56d4c5f6fac5606d4dde23a": {
-			"publicKey": "0x1",
-			"balance": "0xD3C21BCECCEDA1000000",
-			"nonce": "0x1",
-			"class": "0x444",
-			"storage": {
-				"0x1": "0x1",
-				"0x2": "0x2"
-			}
-		},
-		"0x6b86e40118f29ebe393a75469b4d926c7a44c2e2681b6d319520b7c1156d114": {
-			"publicKey": "0x2",
-			"balance": "0xD3C21BCECCEDA1000000"
-		},
-		"0x79156ecb3d8f084001bb498c95e37fa1c4b40dbb35a3ae47b77b1ad535edcb9": {
-			"publicKey": "0x3"
-		},
-		"0x053a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02bf": {
-			"publicKey": "0x4",
-			"balance": "0xD3C21BCECCEDA1000000",
-			"privateKey": "0x115"
-		}
-	},
-	"contracts": {
-		"0x29873c310fbefde666dc32a1554fea6bb45eecc84f680f8a2b0a8fbb8cb89af": {
-			"balance": "0xD3C21BCECCEDA1000000",
-			"class": "0x8",
-			"storage": {
-				"0x1": "0x1",
-				"0x2": "0x2"
-			}
-		},
-		"0xe29882a1fcba1e7e10cad46212257fea5c752a4f9b1b1ec683c503a2cf5c8a": {
-			"balance": "0xD3C21BCECCEDA1000000"
-		},
-		"0x05400e90f7e0ae78bd02c77cd75527280470e2fe19c54970dd79dc37a9d3645c": {
-			"storage": {
-				"0x1": "0x1"
-			}
-		}
-	},
-	"classes": [
-		{
-			"class": "path/to/file/erc20.json",
-			"classHash": "0x8"
-		},
-		{
-			"class": "path/to/file/universal_deployer.json",
-			"classHash": "0x444"
-		},
-		{
-			"class": {
-				"abi": [
-					{
-						"members": [
-							{ "name": "to", "offset": 0, "type": "felt" },
-							{ "name": "selector", "offset": 1, "type": "felt" },
-							{ "name": "data_offset", "offset": 2, "type": "felt" },
-							{ "name": "data_len", "offset": 3, "type": "felt" }
-						],
-						"name": "AccountCallArray",
-						"size": 4,
-						"type": "struct"
-					}
-				],
-                ...
-			}
-		}
-	]
+- `number`: The block number of the genesis block.
+- `parentHash`: The parent hash of the genesis block.
+- `timestamp`: The timestamp of the genesis block.
+- `stateRoot`: The state root of the genesis block.
+- `sequencerAddress`: The sequencer address.
+- `gasPrices`: The gas prices for the L1 tokens at the genesis block.
+- `feeToken`: The network fee token configuration (optional).
+- `universalDeployer`: The universal deployer configuration (optional).
+- `accounts`: The genesis allocations.
+- `contracts`: The genesis contract deployments.
+- `classes`: The classes to declare at genesis.
+
+## Usage Examples
+
+### 1. Creating Genesis Accounts
+
+Genesis accounts are initialized with predefined balances and settings.
```

Comment:
We should add some details on the class hash of default account? In the example the class hash is overridden with `0x444`, could you detail / add info on the effect of that?

Reply:
the examples should be specific. for example,

```md

Creating an account with the address `0x66..` and public key `0x1`. The account will also be allocated `0xD3C21BCECCEDA1000000` amount of the fee token.

"accounts": {
    "0x66efb28ac62686966ae85095ff3a772e014e7fbf56d4c5f6fac5606d4dde23a": {
        "publicKey": "0x1",
        "balance": "0xD3C21BCECCEDA1000000",
    },
}

or 

Declaring a class at path `path/to/file/my_contract.json` with the class hash `0x8`. The class will be declared at genesis.

"classes": [
    {
        "class": "path/to/file/my_contract.json",
        "classHash": "0x8"
    }
]
```

you can try to be more creative with the wording here. dont need to make the explanation too long, try to make it concise and straightforward.